templates/takeovers/_livepatch.html

```diff
@@ -0,0 +1,8 @@
+<section class="row row-hero no-border-bottom">
+    <div class="strip-inner-wrapper">
+        <h1 class="seven-col">Apply critical kernel patches&nbsp;without rebooting</h1>
+        <p class="six-col">The Canonical Livepatch Service for Ubuntu 16.04 LTS reduces downtime while maintaining compliance and security.</p>
+        <img src="{{ ASSET_SERVER_URL }}7a3a71fc-dell-xps-laptop-front.png?w=768" alt="laptop with live patch work" class="for-small six-col row-hero__image" />
```

Comment:
https://assets.ubuntu.com/v1/7a3a71fc-dell-xps-laptop-front.png?w=768 looks pretty pixelated.
Can we try resizing the asset with a better tool and see if it looks better? If so, we can use and upload that.